aikau/src/main/resources/alfresco/lists/views/ListRenderer.js

```diff
@@ -167,6 +186,71 @@ define(["dojo/_base/declare",
        */
       onItemFocused: function alfresco_lists_views_ListRenderer__onItemFocused(evt) {
          this.focusChild(evt.item);
+      },
+
+      /**
+       * This function is called whenever a parent [list]{@link module:alfresco/lists/AlfList} publishes a request
+       * to bring a specfic item into view, e.g. to ensure that a requested item that is displayed off the page is
+       * brought into view by automatically setting the scroll position. This function only checks to see whether or not
+       * the current widget represents that item - it does not perform the actual action to bring the item into view.
+       *
+       * @instance
+       * @param  {object} payload The details of the item to find
+       */
+      onFindItemToBringItemIntoView: function alfresco_lists_views_ListRenderer__onFindItemToBringItemIntoView(payload) {
```

Comment:
This method name is a little confusing. I thought the convention was something similar to "on" + triggering topic?